client/my-sites/plan-features-comparison/header.jsx

```diff
@@ -84,17 +84,28 @@ export class PlanFeaturesComparisonHeader extends Component {
 	}
 
 	getAnnualDiscount() {
-		const { isMonthlyPlan, rawPriceForMonthlyPlan, annualPricePerMonth, translate } = this.props;
+		const { isMonthlyPlan, rawPriceForMonthlyPlan, annualPricePerMonth, translate, planType } =
+			this.props;
 
 		if ( ! isMonthlyPlan ) {
 			const isLoading = typeof rawPriceForMonthlyPlan !== 'number';
 
 			const discountRate = Math.round(
 				( 100 * ( rawPriceForMonthlyPlan - annualPricePerMonth ) ) / rawPriceForMonthlyPlan
 			);
-			const annualDiscountText = translate( `You're saving %(discountRate)s%% by paying annually`, {
-				args: { discountRate },
-			} );
+
+			const annualDiscountText = [
+				'personal-bundle-2y',
+				'value_bundle-2y',
+				'business-bundle-2y',
+				'ecommerce-bundle-2y',
+			].includes( planType )
+				? translate( `You're saving %(discountRate)s%% by paying bi-annually`, {
```

Comment:
I _think_ it's **biennial** here (and no hyphen). As **biannual** is something that happens twice in a year, while **biennial** happens once every two years. 

Not sure which one we use across Calypso/WPcom though, but biennial should be the correct one. 

If the Abacus test is running outside of language=EN, we also need to "string freeze" this, e.g. if it's running in the US (as it also has Spanish and many other language users). 

Approving this. The logic works. So you can just do a quick push with the grammar change, and add string freeze if needed depending on the test config. 👍